net/upnp/src/www/services_upnp.php

```diff
@@ -94,6 +95,12 @@ function miniupnpd_validate_port($port)
             $pconfig[$fieldname] = $config['installedpackages']['miniupnpd']['config'][0][$fieldname];
         }
     }
+
+    // ensure backwards compatibility
+    if (!isset($pconfig['num_permuser'])) {
+      $pconfig['num_permuser'] = 8;
```

Comment:
you can assign this unconditionally before the copy loop. if it's there it will be overwritten. the only problem you're going to have is now that value will stick to the config.xml forever (even if it's 8)

Reply:
I went with @AdSchellevis's suggestion above and removed it entirely.  It will still have eight fields displayed if there's nothing set, and won't add the value to the `config.xml` file unless somebody explicitly sets it.

Reply:
sounds good, thanks!